test/build/yarn-workspaces/index.test.js

```diff
@@ -0,0 +1,15 @@
+const fs = require('fs');
+const path = require('path');
+const execa = require('execa');
+
+it('yarn workspaces can build', () => {
+  // run snowpack build on a Yarn workspaces project
+  execa.sync('npm', ['run', 'TEST'], {cwd: path.resolve(__dirname, 'packages', 'snowpack')});
+
+  const builtFile = fs.existsSync(
+    path.resolve(__dirname, 'packages', 'snowpack', 'build', '_dist_', 'index.js'),
+  );
+
+  // expect the built file to exist (build didn’t fail with “missing dep”, etc.)
+  expect(builtFile).toBe(true);
```

Comment:
once this is merged, lets build that test runner. Exists checks are a great start, but we should really be comparing the actual build directory result against an "expected" snapshot in memory. You should be able to copy a lot of our work from `test/integration/runner`.